packages/contracts/contracts/ux/CreateNewsroomInGroup.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.4.23;
+
+import "../newsroom/NewsroomFactory.sol";
+import "../proof-of-use/groups/UserGroups.sol";
+
+contract CreateNewsroomInGroup {
+  NewsroomFactory factory;
+  UserGroups groups;
+
+  constructor(NewsroomFactory _factory, UserGroups _groups) {
+    factory = _factory;
+    groups = _groups;
+  }
+
+  function create(string name, string charterUri, bytes32 charterHash, address[] initialOwners, uint initialRequired)
```

Comment:
This is so that you can create the newsroom and multisig in network in one transaction?  How much more does it cost then doing it in two transactions?